core/src/test/scala-2/caliban/Scala2SpecificSpec.scala

```diff
@@ -39,6 +43,30 @@ object Scala2SpecificSpec extends DefaultRunnableSpec {
             """{"events":[{"organizationId":7,"title":"Frida Kahlo exhibition"}],"painters":[{"name":"Claude Monet","movement":"Impressionism"}]}"""
           )
         )
+      },
```

Comment:
If the test is same for Scala 2 and Scala 3, you can simply add it in the `scala` folder, no need to duplicate it. These files are just for Scala2 or Scala3 specific features (like Scala 3 enums).

Reply:
For some reason I though it had to be there for it to run for each version - I'll move the test.